element_interface/utils.py

```diff
@@ -140,31 +145,57 @@ def ingest_csv_to_table(
         )
         if verbose:
             insert_len = len(table) - prev_len
-            print(
+            logger.info(
                 f"\n---- Inserting {insert_len} entry(s) "
                 + f"into {table.table_name} ----"
             )
 
 
-def recursive_search(key: str, dictionary: dict) -> any:
-    """Return value for key in a nested dictionary
-
-    Search through a nested dictionary for a key and returns its value.  If there are
-    more than one key with the same name at different depths, the algorithm returns the
-    value of the least nested key.
+def str_to_bool(value) -> bool:
+    """Return whether the provided string represents true. Otherwise false.
 
     Args:
-        key (str): Key used to search through a nested dictionary
-        dictionary (dict): Nested dictionary
+        value (any): Any input
 
     Returns:
-        value (any): value of the input argument `key`
+        bool (bool): True if value in ("y", "yes", "t", "true", "on", "1")
+    """
+    # Due to distutils equivalent depreciation in 3.10
+    # Adopted from github.com/PostHog/posthog/blob/master/posthog/utils.py
```

Comment:
```suggestion
```

Reply:
I think it is fair to mention this within a pull request comment or the Concepts page, but I don't see a need to keep this comment in the codebase.